client/client.go

```diff
@@ -326,13 +326,27 @@ func decodeInto(reader io.Reader, v interface{}) error {
 	return nil
 }
 
-// doSync performs a request to the given path using the specified HTTP method.
+// RequestInfo holds the information to perform a request to the daemon.
+type RequestInfo struct {
```

Comment:
Is there a reason not to call this `Request` ?

Reply:
I agree `Request` is nicer, except for the existing `ResultInfo` type has the `Info` suffix, so it's probably better to match that for consistency.

Reply:
As @benhoyt said, it's `RequestInfo` for consistency so I'm resolving this. Feel free to unresolve if you'd like further discussion (:

Reply:
ResultInfo doesn't have the Result. It is a placeholder. I would prefer Request.